daml-lf/spec/value.rst

```diff
@@ -461,49 +457,44 @@ As of version 6, these fields are included:
 
   [A-Za-z0-9._:-]+
 
-If ``relative`` is unset or false, this is an absolute contract ID;
```

Comment:
what happened to this? you still include the boolean field above. If we got rid of relative contract ids, don’t we need to kill both?

Reply:
We should have kill that for some time. 
We always set to `false` and never read it. 
It kill it in this PR.